test/actions/Test_Actions.php

```diff
@@ -1498,5 +1498,46 @@ public function cannot_set_empty_actionid()
         // An empty ActionID
         $action->setActionID('');
     }
+
+    /**
+     * @test
+     * @expectedException \PAMI\Exception\PAMIException
+     */
+    public function cannot_set_updateconfig()
```

Comment:
Perhaps you can make a specific can_update_config test where you send 2 or 3 UpdateConfig actions and assert that the sequence sent is indeed increasing (like in [can_originate](https://github.com/marcelog/PAMI/pull/60#discussion-diff-23936682R1420)).. makes sense?